### PR TITLE
add docstrings, error handling and more detailed information in JSON response

### PR DESCRIPTION
**Pull request by AtanasKisyov**

### CODE REVIEW

File: Carwash/orders/views.py [hunk 1]
```diff
@@ -1,9 +1,71 @@
...
+            'quantity_of_places': car_wash.quantity_of_places,
+        }
+
+    def retrieve(self, request, *args: tuple, **kwargs: dict) -> serializer_class:
+        """
+        Re-write default retrieve method to alter the returned data
+        """
+
+        data = super().retrieve(request, *args, **kwargs)
+        data.data['user'] = self._alter_user_information(data)
```
Comment (strangerok1507) [inline Carwash/orders/views.py]: you don't need it.
use nested serializers for it https://www.django-rest-framework.org/api-guide/relations/#nested-relationships
in create/update request you should send only id of object

**Review by strangerok1507 — CHANGES_REQUESTED** [review]

File: Carwash/orders/views.py (hunk 1 above)
Comment (AtanasKisyov) [reply Carwash/orders/views.py]: I added a check in __init__ to see if request method is get, set depth in Meta class to 2 and related fields to read only

File: Carwash/orders/utils.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+from rest_framework.views import exception_handler
```
Comment (strangerok1507) [inline Carwash/orders/utils.py]: you don't need this file

**Review by strangerok1507 — APPROVED** [review]
remove utils py and can merge

**Pull request merged**